backend/src/BuiltinDarkInternal/Libs/Documentation.fs

```diff
@@ -1,4 +1,6 @@
 /// Builtin functions for documentation
+///
+/// TODO: consider moving this from DarkInternal to a publicly-accessible space
```

Comment:
Should users be allowed to call `@Darklang.Documentation.allFunctions`? maybe not immediately useful but seems innocent to allow, to me.

Reply:
They should not. I would say the opposite: the less surface area to users the better.